projects/core/src/lib/components/filter/filter-list-item/filter-list-item.component.ts

```diff
@@ -22,7 +22,19 @@ export class FilterListItemComponent {
 
   @Input()
   public set filterGroup(filterGroup: ExtendedFilterGroupModel | null) {
-    this.filter = filterGroup;
+    if (!filterGroup) {
+      return;
+    }
+    this.filter = {
+      ...filterGroup,
+      filters: filterGroup?.filters.filter(f => {
+        if (FilterTypeHelper.isAttributeFilter(f) && f.attributeNotFound) {
+          console.error(`Filtered attribute '${f.attribute}' not found. Filter hidden and disabled.`);
```

Comment:
```suggestion
          console.error(`Attribute '${f.attribute}' not found. Filter hidden and disabled.`);
```
Attributes are 'hidden', not filtered.. Attributes could also have been removed or renamed.